datacite-doi-commons/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+    api group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.0'
+    api group: 'com.github.BIBSYSDEV.nva-doi-partner-data', name: 'datacite-transform', version: '0.3'
+    api group: 'com.github.BIBSYSDEV.nva-publication-api', name: 'publication-doi-commons', version: '0.1.13'
+
```

Comment:
explicit add the `assign-doi-datacite` dependency here. Don't want the client in `publication-doi-comments` .